SlaveWAMPServer.js

```diff
@@ -91,31 +107,64 @@ class SlaveWAMPServer extends WAMPServer {
 		}
 	}
 
+	/**
+	 * @param {string} socketId
+	 * @returns {boolean}
+	 * @returns {undefined}
+	 */
 	onSocketDisconnect(socketId) {
 		return delete this.interProcessRPC[socketId];
 	}
 
-	getCall(request) {
-		return get(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, false);
+	/**
+	 * @param {InterProcessRPCRequestSchema}[request={}] request
+	 * @param {Function} cb
+	 * @returns {undefined}
+	 */
+	saveCall(request = {}, cb) {
+		if (!request.socketId || !request.procedure || !request.signature) {
+			throw new Error('Cannot save a call for wrong InterProcessRPCRequest request');
+		}
+		if (!cb) {
+			throw new Error('Cannot save a call without callback');
+		}
+		return setWith(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, cb, Object);
 	}
 
-	saveCall(request, cb) {
-		return setWith(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, cb, Object);
+	/**
+	 * @param {InterProcessRPCRequestSchema}[request={}] request
+	 * @returns {InterProcessRPCRequestSchema|false}
+	 */
+	getCall(request = {}) {
+		return get(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, false);
 	}
 
-	deleteCall(request) {
+
+	/**
+	 * @param {InterProcessRPCResponseSchema}[request={}] request
+	 * @returns {boolean}
+	 */
+	deleteCall(request = {}) {
```

Comment:
Consider destructuring `socketId`, `procedure`, and `signature` out immediately here for ease of reading in the function body.